### PR TITLE
created model and endpoint for buyer search function

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by karapeoples — COMMENTED** [review]
I did try to resolve the conflicts but it must have created other issues since tests are failing now.

File: api/items/itemsRouter.js [hunk 1]
```diff
@@ -33,10 +35,30 @@ router.get('/:itemID', authRequired, async (req, res) => {
...
 
+// GET all items
+router.get('/', async (req, res) => {
+  const { itemID } = req.params;
+  const response = await Model.getAllItemInfo('item', itemID);
+  try {
+    if (response) {
+      res
+        .status(200)
+        .json({ message: `Retrieved all available items` });
```
Comment (sam-h-hall) [inline api/items/itemsRouter.js]: Don't you need to return the response object?

**Review by sam-h-hall — DISMISSED** [review]

**Review by sam-h-hall — DISMISSED** [review]
Everything looks complete, nice work

**Review by jonathan-portillo — COMMENTED** [review]
Looks like checks are still failing, looks like its an issue with the formatting. We might need to look into that before we're even able to merge it into main.

**Review by jonathan-portillo — APPROVED** [review]
Pretty straight forward and looks like you fixed the eslint issue. Keep it up!!

**Review by Brodt7258 — APPROVED** [review]

**Review by sam-h-hall — APPROVED** [review]
Looks great! Nice job

**Pull request merged**